tests/controllers/beneficiary.test.js

```diff
@@ -0,0 +1,123 @@
+const {connectDatabase, closeDatabase, clearDatabase} = require('../common');
+const {Vendor, User, Project, Beneficiary} = require('../../modules');
+const {
+    TokenDistributionModel
+} = require('../../modules/models');
+
+const userData = {
+    name: 'Manjik',
+    email: 'man@gmail.com',
+    phone: '9841602388',
+    wallet_address: '0x7c0179776BB143a36C9d338F3Fa6149F40BaAc30',
+    agency: '5f770576880d16420eaedefb'
+};
+
+var projectPayload = {
+
+    name: "Flood Collection",
+    end_date: "2023-01-03",
+    location: "kathmandu",
+    description: "Rescue mission"
+
+}
+
+let currentUser;
+    var payload = {
+        name: "Disabled Beneficiary",
+        phone: "9097879",
+        wallet_address: "0x7c0179776BB143a36C9d338F3Fa6149F40BaAc30",
+        email: "san@gamil.com",
+        address: "kathmandu",
+        address_temporary: "kathmandu",
+        gender: 'M',
+        govt_id: "78945",
+        agency: '5f770576880d16420eaedefb',
+        projects: "5f6b2f815685931cbfe4dad8"
+
+    }
+
+
+
+
+
+describe('Beneficiary CRUD', () => {
+    let benf;
+    let token
+    beforeAll(async () => {
+        await connectDatabase();
+        currentUser = await User.create(userData);
+        payload.currentUser = currentUser;
+        projectPayload.currentUser = currentUser;
+        projectPayload.project_manager = currentUser.id;
+        project = await Project.add(projectPayload);
+        payload.projects = project.id;
+        token= TokenDistributionModel(payload);
+    });
+    afterAll(async () => {
+        await closeDatabase();
+    });
+
+    it ('beneficiary can be created correctly', async() => {
```

Comment:
Can you also add the test for bulk beneficiary addition. You can have seperate json for the data but It would be better if you use some packages like faker for random data generation

Reply:
completed

Reply:
Did you write tests for addMany function in beneficiary Controller?